AmplicaLabs/social-app-template
# Consolidate env files into docker compose files for bare-metal and docker configs

# Purpose
The goal of this PR is to implement a docker compose file that launches all the containers required to run the social-app-template-backend locally (bare metal) and also with the social-app-template-backend running the in docker.

## Solution
- Use map objects to partition environment variables for each container
- Add `docker-compose-bare-metal.yaml`
- Update `docker-compose.yaml` to use internal env variables

Developed with:
@JoeCap08055 @claireolmstead 

## Steps to Verify
1. Execute `docker compose -f docker-compose-bare-metal.yaml up -d` or `restart-chain-bare-metal.sh`
2. `npm run start:dev`
3. From /frontend: `npm start`
4. Verify that `social-app-template` works for all currently supported features.
5. Execute `restart-chain-docker.sh`  and verify currently supported features.
